provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -56,51 +56,62 @@ def _create_pool_descriptions(self) -> List[PoolDesc]:
 
         for x in range(storage_set_count):
             storage_set_name = conf.get(
-                f'cluster>{cluster_id}>storage_set{x+1}>name')
+                f'cluster>{cluster_id}>storage_set[{x}]>name')
 
             data_devices_count: int = 0
             for node in conf.get(
-                    f'cluster>{cluster_id}>storage_set{x+1}>server_nodes'):
+                    f'cluster>{cluster_id}>storage_set[{x}]>server_nodes'):
                 data_devices_count += len(
-                    conf.get(f'server_node>{node}>cvg[0]>data_devices'))
-
-            data_units_count = int(conf.get(
-                f'cluster>{cluster_id}>storage_set{x+1}>durability>data'))
-            parity_units_count = int(conf.get(
-                f'cluster>{cluster_id}>storage_set{x+1}>durability>parity'))
-            spare_units_count = int(conf.get(
-                f'cluster>{cluster_id}>storage_set{x+1}>durability>spare'))
+                    conf.get(
+                        f'server_node>{node}>storage>cvg[0]>data_devices'))
+
+            data_units_count = int(
+                conf.get(
+                    f'cluster>{cluster_id}>storage_set[{x}]>durability>data'))
+            parity_units_count = int(
+                conf.get(
+                    f'cluster>{cluster_id}>storage_set[{x}]>durability>parity')
+            )
+            spare_units_count = int(
+                conf.get(
+                    f'cluster>{cluster_id}>storage_set[{x}]>durability>spare'))
 
             min_pool_width = data_units_count + parity_units_count \
                 + spare_units_count
-            if (data_devices_count != 0) and not (
-                    data_devices_count >= min_pool_width):
+            if (data_devices_count !=
```

Comment:
I get flake errors in this case as `line break after an operator` did the build work for you?

Reply:
Yes, build is passing.

Reply:
In fact even if we move that operator to next light exact opposite error will be displayed(Line break occurred before a binary operator)

Reply:
Yeah, make rpm works but if you do make in `provisioning/miniprov` you will see the errors. Just make sure that `make` in `provisioning/miniprov`.